app/controllers/concerns/pagination.rb

```diff
@@ -3,9 +3,13 @@
 # Default options and configuration for pagination
 module Pagination
   extend ActiveSupport::Concern
+
+  ParamType = ActionController::Parameters
```

Comment:
Nit/optional: I wonder if we should go ahead and move this to the top of the application_controller.rb since we're doing it here and in `concerns/parameters.rb`.